config/god/hets_workers.rb

```diff
@@ -6,8 +6,14 @@ def group
   end
 
   def start_cmd
-    hets_opts = ' ' << hets_server_options.join(' ')
-    'exec nice hets -X' << hets_opts
+    hets_opts = ' -X ' << hets_server_options.join(' ')
+    hets_server = Array(HetsSettings['hets_path']).flatten
+      .map { |path| File.expand_path path }.find { |path| File.exists?(path) }
```

Comment:
Prefer `detect` over `find`.<br>Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.